docs/HyperIndex/migration-guide.md

```diff
@@ -0,0 +1,34 @@
+---
+id: migration-guide
+title: Migration Guide
+sidebar_label: Migrate to HyperIndex
+slug: /migration-guide
+---
+
+# HyperIndex Migration Guide
```

Comment:
It'd be nice to include a TheGraph to the title. I think it's good for SEO and search engine